tutorials/evals/evaluate_code_functionality_classifications.ipynb

```diff
@@ -0,0 +1,2511 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    import os
</pre><p>same here: update your environment so your username isn't exposed in this notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/2802/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>